javascript/sdk/lib/types.ts

```diff
@@ -417,3 +417,9 @@ export interface BlockResponse {
   already_reported?: boolean;
   ioc_id?: string;
 }
+
+export interface GeneratedWebhook {
```

Comment:
s1 returns that Header dict too

Reply:
@makk94 fixed :D

Reply:
does it need a `?` or something since crowdstrike doesnt return it?